themes/startwords/layouts/article/single.html

```diff
@@ -55,24 +55,25 @@ <h1>{{ .Params.Title }}</h1>
                 {{- end }}
                 </p>
             </header>
+
+            {{/* elements for PDF header & footer */}}
+            <section class='print-only'>
+
+                <a class="first-page-header" href="{{ relURL "/" }}" aria-label="Startwords">
+                    <img alt="Startwords" src="{{ relURL "/pdf-logotype.svg" }}" /></a>
+
+                <a class="page-header" href="{{ relURL "/" }}" aria-label="Startwords">
+                    <img alt="Startwords" src="{{ relURL "/pdf-logo.svg" }}" /></a>
+
+                 <a href="http://doi.org/{{ .Params.doi }}" rel="alternate" class="page-doi">doi:{{ .Params.doi }}</a>
+
+            </section>
+
```

Comment:
worth moving this into its own template partial? or is there no point

Reply:
I don't think so? Not used anywhere else right now